### PR TITLE
Add support of ping the web driver

### PR DESCRIPTION
**Pull request by maidmaid**
It happens sometimes that we lose the connection with the web driver for many different reasons. Some other times, I'm just asking myself if this connection is still alive. A "ping" helper could be great just for checking this connection, in a debugging purpose.

### CODE REVIEW

File: src/Client.php [hunk 1]
```diff
@@ -433,4 +434,44 @@ public function getMouse()
...
     }
+
+    /**
+     * Checks the web driver connection (and logs "pong" into the DevTools console).
+     *
+     * @param int $timeout sets the connection/request timeout in ms
+     *
+     * @return bool true if connected, false otherwise
+     */
+    public function ping($timeout = 1000)
```
Comment (dunglas) [inline src/Client.php]:
```suggestion
    public function ping(int $timeout = 1000): bool
```

**Comment by maidmaid** [comment]
Tests added and changes done :)

File: src/Client.php [hunk 2]
```diff
@@ -433,4 +434,44 @@ public function getMouse()
...
+            } else {
+                $this->webDriver->findElement(WebDriverBy::tagName('html'));
+            }
+        } catch (\Throwable $t) {
+            return false;
+        } finally {
+            if ($this->webDriver instanceof RemoteWebDriver) {
+                $this->webDriver->getCommandExecutor()
+                    ->setConnectionTimeout(30000)
+                    ->setRequestTimeout(30000)
```
[the remark this answers is not in the document]
Comment (maidmaid) [reply src/Client.php]: There aren't public getters for that. Do you have a suggestion ?

[the remark this answers is not in the document]
Comment (dunglas) [reply src/Client.php]: can we move forward on this @stof?

[the remark this answers is not in the document]
Comment (stof) [reply src/Client.php]: there is no public setter, but there is a public getter for the WebDriver instance, so they can change what they want.

**Comment by dunglas** [comment]
Thanks @maidmaid

**Pull request merged**